test_task.py

```diff
@@ -0,0 +1,75 @@
+
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):
+    """Retrieves {'abbreviation': (name, team)}" format dict from abbreviations.txt"""
+    abbreviations_dict = {}
+    with open(file_name, 'r') as fn:
+        for line in fn:
+            matchObj = re.match(r'^(\w+)_([a-zA-Z\s]+)_([a-zA-Z\s]+)$', line)
```

Comment:
You can unpack values here this way:
abbreviation, name, team = re.match(r'^(\w+)_([a-zA-Z\s]+)_([a-zA-Z\s]+)$', line).groups()
And then 
abbreviations_dict[abbreviation] = (name, team.rstrip())